src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -2577,24 +2592,67 @@ public void testLineSplitWithVoltageLevel() {
         assertEquals(lineSplitWoVLasJSON, updateRequest.get().getBody());
 
         webTestClient.post()
-            .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/line-splits",
-                studyNameUserIdUuid, modificationNodeUuid)
-            .bodyValue("bogus")
-            .exchange()
-            .expectStatus().is5xxServerError()
-            .expectBody(String.class).isEqualTo("400 BAD_REQUEST");
+                .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/line-splits",
+                        studyNameUserIdUuid, modificationNodeUuid)
+                .bodyValue("bogus")
+                .exchange()
+                .expectStatus().is5xxServerError()
+                .expectBody(String.class).isEqualTo("400 BAD_REQUEST");
 
         webTestClient.put()
-            .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{modificationUuid}/line-splits",
-                studyNameUserIdUuid, modificationNodeUuid, MODIFICATION_UUID)
-            .bodyValue("bogus")
-            .exchange()
-            .expectStatus().is5xxServerError()
-            .expectBody(String.class).isEqualTo("400 BAD_REQUEST");
+                .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{modificationUuid}/line-splits",
+                        studyNameUserIdUuid, modificationNodeUuid, MODIFICATION_UUID)
+                .bodyValue("bogus")
+                .exchange()
+                .expectStatus().is5xxServerError()
+                .expectBody(String.class).isEqualTo("400 BAD_REQUEST");
 
         requests = getRequestsWithBodyDone(2);
     }
 
+    @SneakyThrows
+    @Test
+    public void testLineAttachToVoltageLevel() {
+        createStudy("userId", CASE_UUID);
+        UUID studyNameUserIdUuid = studyRepository.findAll().get(0).getId();
```

Comment:
useless : createStudy returns the study uuid

Reply:
Corrected